src/containers/Tablet/Tablet.tsx

```diff
@@ -98,57 +100,69 @@ export const Tablet = () => {
-            <div className={b('pane-wrapper')}>
-                <div className={b('left-pane')}>
-                    <ul className={b('links')}>{externalLinks.map(renderExternalLinks)}</ul>
-                    <div className={b('row', {header: true})}>
-                        <span className={b('title')}>{i18n('tablet.header')}</span>
-                        <EntityStatus status={Overall} name={TabletId} />
-                        <a
-                            rel="noopener noreferrer"
-                            className={b('link', {external: true})}
-                            href={`${backend}/tablets?TabletID=${TabletId}`}
-                            target="_blank"
-                        >
-                            <Icon name="external" />
-                        </a>
-                        {Leader && <Tag text="Leader" type="blue" />}
-                        <span className={b('loader')}>{loading && <Loader size="s" />}</span>
+        return (
+            <div className={b()}>
+                <div className={b('pane-wrapper')}>
+                    <div className={b('left-pane')}>
+                        <ul className={b('links')}>{externalLinks.map(renderExternalLinks)}</ul>
+                        <div className={b('row', {header: true})}>
+                            <span className={b('title')}>{i18n('tablet.header')}</span>
+                            <EntityStatus status={Overall} name={TabletId} />
+                            <a
+                                rel="noopener noreferrer"
+                                className={b('link', {external: true})}
+                                href={`${backend}/tablets?TabletID=${TabletId}`}
+                                target="_blank"
+                            >
+                                <Icon name="external" />
+                            </a>
+                            {Leader && <Tag text="Leader" type="blue" />}
+                            <span className={b('loader')}>{loading && <Loader size="s" />}</span>
+                        </div>
+                        <TabletInfo tablet={tablet} tenantPath={tenantName} />
+                        <TabletControls tablet={tablet} fetchData={fetchData} />
+                    </div>
+                    <div className={b('rigth-pane')}>
+                        <TabletTable history={history} />
                     </div>
-                    <TabletInfo tablet={tablet} tenantPath={tenantName} />
-                    <TabletControls tablet={tablet} fetchData={fetchData} />
-                </div>
-                <div className={b('rigth-pane')}>
-                    <TabletTable history={history} />
                 </div>
             </div>
-        </div>
+        );
+    };
+    return (
+        <>
+            <Helmet>
+                <title>{`${id} - ${
+                    tenantName || queryClusterName || i18n('tablet.header')
+                }`}</title>
```

Comment:
For me `i18n('tablet.header')` is excessive here. It is not an alternative to tenant or cluster name if they are missing. For me it's more like `TabletId - YDB Monitoring` or `TabletId - Cluster - YDB Monitoring` in this case. Or if `TabletId` is not obvious indicator, something like `TabletId - Tablet - TenantName / ClusterName - YDB Monitoring` (`Tablet` is always present)